File: modulo1/segmento3/apunte6/Ejercicio2.java
import java.util.Scanner;

public class Ejercicio2 {
	public static void main(String args[]){
		Scanner keyboard = new Scanner(System.in);

			String otro = "n";

				do{

				System.out.print("Ingrese primer numero a sumar: ");
				Integer num1 = keyboard.nextInt();

				System.out.print("Ingrese segundo numero a sumar : ");
				Integer num2 = keyboard.nextInt();
				Integer result = num1 + num2;

				System.out.println("El resultado es: " + result);

				System.out.println("==================================");
				
				System.out.println("Desea hacer otra suma (s/n) : ");
				otro = keyboard.next();

			} while(otro.equals("s"));
			
	}
}

File: modulo1/segmento3/apunte6/Ejercicio2.web/ejercicio2.js
function ejercicio2() {
	let otro = 'si';

	while(otro == 'si') {
		alert("Ingrese 2 numeros para sumar");
        num1 = parseInt(prompt('Ingrese un numero: '));
        num2 = parseInt(prompt('Ingrese otro numero: '));
        let suma = num1 + num2 ;
        alert (suma);
		otro = prompt('Desea realizar otra suma ( si/no ): ');
	}
}